quinn-proto/Cargo.toml

```diff
@@ -17,8 +17,9 @@ all-features = true
 maintenance = { status = "experimental" }
 
 [features]
-default = ["tls-rustls", "log"]
+default = ["tls-rustls", "rustls-ring", "log"]
 tls-rustls = ["rustls", "ring"]
+rustls-ring = ["rustls/ring"]
```

Comment:
Document the use case for this? It allows downstream code using some other rustls backend to avoid a ring dependency, right?